### PR TITLE
add apm-server url to handlebars tmpl

### PR DESCRIPTION
**Pull request by stuartnelson3**
## Motivation/summary

Display a publicly-reachable url for apm-server when running in cloud or eck

closes https://github.com/elastic/apm-server/issues/4895

## Checklist

- [x] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)

## How to test these changes

1. Start apm-server under fleet
2. Go to the apm-server integration in the fleet ui
3. Verify that the url is visible

### CODE REVIEW

**Review by axw — APPROVED** [review]

**Comment by jalvz** [comment]
shouldn't we be rolling version 0.3.0 ?

**Comment by axw** [comment]
@jalvz we should be bumping to 7.14.0: https://github.com/elastic/apm-server/issues/4898. That can happen independently of this change; 0.2.0 is on the 7.13 branch.

**Comment by axw** [comment]
@stuartnelson3 can you please remove apmpackage/apm/0.2.0? That version lives on the 7.13 branch.

Slightly longer term I think we should probably remove the version from the path in the apm-server repo, and only add it when copying to package storage.

**Comment by stuartnelson3** [comment]
removed :ok_hand:

File: apmpackage/apm/0.3.0/manifest.yml [hunk 1]
```diff
@@ -47,6 +47,12 @@ policy_templates:
             required: true
             show_user: true
             default: localhost:8200
+          - name: url
+            type: text
+            title: URL
+            required: true
+            show_user: true
+            default: localhost:8200
```
Comment (axw) [inline apmpackage/apm/0.3.0/manifest.yml:55]:
```suggestion
            default: http://localhost:8200
```

?

**Review by axw — APPROVED** [review]

File: apmpackage/apm/0.3.0/manifest.yml (hunk 1 above)
Comment (stuartnelson3) [reply apmpackage/apm/0.3.0/manifest.yml:55]: I remember someone saying that it should default to the same value as `host`, but I'm happy to add the `http://`. Not sure what the "?" is asking :)

Comment (simitt) [reply apmpackage/apm/0.3.0/manifest.yml:55]: @stuartnelson3 I mentioned to default to the `host` value, but am +1 on adding `http://`

Comment (axw) [reply apmpackage/apm/0.3.0/manifest.yml:55]: that was vague way of saying I'm not 100% sure, but I think it should be a proper URL

**Review by simitt — APPROVED** [review]

**Comment by jalvz** [comment]
> Slightly longer term I think we should probably remove the version from 
the path in the apm-server repo, and only add it when copying to package storage

we are runing `elastic-package` thou, which expects a version folder.

**Comment by axw** [comment]
@jalvz which command specifically?

If I run `elastic-package lint` in `apmpackage/apm` I get:

```
int the package
Error: package root not found
```

If I run in the `apmpackage/apm/0.3.0` I get:

```
Lint the package
Done
```

If I rename "0.3.0" to "foobar", it still works; `elastic-package lint` doesn't appear to be traversing up to find a version directory.

**Comment by axw** [comment]
I've observed the URL showing up in the integration editor UI, but I think we should probably test this in ESS/ECE.

**Comment by simitt** [comment]
Tested on ESS with with latest `7.14` snapshot - URL shows up in the UI.
Screenshot 2021-07-12 at 11 11 51

**Pull request merged**